client/verta/verta/operations/monitoring/monitored_entity.py

```diff
@@ -86,24 +87,24 @@ def workspace(self):
     def alerts(self):
         return Alerts(self._conn, self._conf, self.id)
 
+    @property
+    def summaries(self):
+        return Summaries(self._conn, self._conf)
+
```

Comment:
I'm guessing this is the starting point for that other ticket about moving summary CRUD to ME 👌